src/components/Column/ColumnSettings.tsx

```diff
@@ -36,109 +40,66 @@ export const ColumnSettings: FC<ColumnSettingsProps> = ({id, name, color, visibl
+            onClick: () => {
               onClose?.();
-              dispatch(Actions.deleteColumn(id));
-            }}
-          >
-            <Trash />
-            {t("Column.deleteColumn")}
-          </button>
-        </li>
-        <li>
-          <button
-            aria-label="Backlog Blue"
-            title="Backlog Blue"
-            className={classNames(getColorClassName("backlog-blue"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "backlog-blue", index, visible}))}
-          />
-          <button
-            aria-label="Planning Pink"
-            title="Planning Pink"
-            className={classNames(getColorClassName("planning-pink"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "planning-pink", index, visible}))}
-          />
-          <button
-            aria-label="Poker Purple"
-            title="Poker Purple"
-            className={classNames(getColorClassName("poker-purple"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "poker-purple", index, visible}))}
-          />
-          <button
-            aria-label="Value Violet"
-            title="Value Violet"
-            className={classNames(getColorClassName("value-violet"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "value-violet", index, visible}))}
-          />
-          <button
-            aria-label="Goal Green"
-            title="Goal Green"
-            className={classNames(getColorClassName("goal-green"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "goal-green", index, visible}))}
-          />
-          <button
-            aria-label="Yielding Yellow"
-            title="Yielding Yellow"
-            className={classNames(getColorClassName("yielding-yellow"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "yielding-yellow", index, visible}))}
-          />
-          <button
-            aria-label="Online Orange"
-            title="Online Orange"
-            className={classNames(getColorClassName("online-orange"), "column__color-button")}
-            onClick={() => dispatch(Actions.editColumn(id, {name, color: "online-orange", index, visible}))}
-          />
-        </li>
-      </ul>
+              dispatch(Actions.editColumn(id, {name, color, index, visible: !visible}));
+            },
+          },
+          {
+            label: t("Column.editName"),
+            icon: <Edit />,
+            onClick: () => {
+              onNameEdit?.();
+              onClose?.();
+            },
+          },
+          {
+            label: t("Column.resetName"),
+            icon: <Close />,
+            onClick: () => (setOpenColumnSet ? setOpenColumnSet((o) => !o) : () => {}),
+          },
+        ]}
```

Comment:
I think it could be beneficial if you extract the items array to a constant. I think it would improve the readability and make the JSX less bloated